WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -61,11 +61,16 @@
 	 * 	'lambda' => array(
 	 * 		'type'      => 'error' | 'warning',
 	 * 		'message'   => 'Use anonymous functions instead please!',
-	 * 		'functions' => array( 'eval', 'create_function' ),
+	 * 		'functions' => array( 'eval', 'create_function', 'mysql_*' ),
+	 *      // Only useful when using wildcards:
```

Comment:
Good catch, I hadn't noticed. It should be all spaces as it's in a comment, but the earlier code wasn't. Fixed now.